bundle/manifests/opendatahub-operator.clusterserviceversion.yaml

```diff
@@ -72,7 +72,7 @@ metadata:
     categories: AI/Machine Learning, Big Data
     certified: "False"
     containerImage: quay.io/opendatahub/opendatahub-operator:v2.1.0
-    createdAt: "2023-10-30T14:38:57Z"
+    createdAt: "2023-8-23T00:00:00Z"
```

Comment:
?

Reply:
Don't ask... It's what operator SDK produces

Reply:
It's essentially the same as in the `main` at this moment: https://github.com/opendatahub-io/opendatahub-operator/blob/6acf1a9c241b642d23a6cc4d9aeb3f8d6e12be35/bundle/manifests/opendatahub-operator.clusterserviceversion.yaml#L75

Reply:
Do you have to update it in the patch? I doubt.